lib/cms/processFieldsJs.ts

```diff
@@ -98,20 +96,11 @@ async function fieldsArrayToJson(fields: Array<FieldsJs>): Promise<string> {
 }
 
 /**
- * Takes in a path to a javascript file and either dynamically imports it or requires it, and returns, depending on node version.
+ * Takes in a path to a javascript file and dynamically imports it.
  * @param {string} filePath - Path to javascript file
- * @returns {Promise | undefined} - Returns _default_ exported content if ESM, or exported module content if CJS, or undefined if node version < 13.2 and file is .mjs.
+ * @returns {Promise} - Returns _default_ exported content if ESM, or exported module content if CJS.
  */
 async function dynamicImport(filePath: string): Promise<any> {
-  if (semver.gte(process.version, '13.2.0')) {
```

Comment:
We should test this but I don't think any of this code was necessary because we no longer support this node version

Reply:
Yeah this is ok to remove if this is going to be a major release anyways